### PR TITLE
docs: Update Terminology Glossary page

### PR DESCRIPTION
**Pull request by anithamuthu2022**
[DEV-2963]

[DEV-2963]: https://heartex.atlassian.net/browse/DEV-2963?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/source/guide/index.md [hunk 1]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
-| Annotations | The output of a labeling task. Previously called "completions". |
-| Templates | Example labeling configurations that you can use to specify the type of labeling that you're performing with your dataset. See [all available templates](/templates) |
-| Tags | Configuration options to customize the labeling interface. See [more about tags](/tags). |
+When you upload data to Label Studio, each item in the dataset becomes a labeling task. The following table defines the terms that appear throughout Label Studio.
+
+**Annotation**
+Annotation is the process of labeling data in various formats such as video, images, or text, so that machines can understand it. The output of a labeling task which was previously called **Completions**. 
+
+**Artificial Intelligence (AI)**
+AI is the intelligence demonstrated by machines, as opposed to the natural intelligence displayed by animals including humans.  It refers to the simulation of human intelligence in machines that are programmed to think like humans and mimic their actions.
```
Comment (anithamuthu2022) [inline docs/source/guide/index.md:62]: AI is the intelligence demonstrated by machines, as opposed to the natural intelligence displayed by animals including humans. It refers to the simulation of human intelligence in machines that are programmed to think like humans and mimic their actions. Label Studio uses [API-powered sentiment analysis](https://labelstud.io/blog/understanding-sentiment-analysis/) to make invisible trends visible, and highly complex, manual tasks more manageable.

File: docs/source/guide/index.md [hunk 2]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+
+**Audio classification**
```
Comment (anithamuthu2022) [inline docs/source/guide/index.md:64]: [Audio classification](/audio_classification.html) or sound classification can be referred to as the process of analyzing audio recordings. This amazing technique has multiple applications in the field of AI and data science such as chatbots, automated voice translators, virtual assistants, music genre identification, and text-to-speech applications. To perform audio classification tasks on specific segments of audio clips, use [the interactive template preview](audio_regions.html#Interactive-Template-Preview) to listen to an audio file and classify the topic of the clip.

File: docs/source/guide/index.md [hunk 3]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Audio classification or sound classification can be referred to as the process of analyzing audio recordings. This amazing technique has multiple applications in the field of AI and data science such as chatbots, automated voice translators, virtual assistants, music genre identification, and text to speech applications.
+
+**Audio labeling**
```
Comment (anithamuthu2022) [inline docs/source/guide/index.md:67]: [Audio labeling](/audio_regions.html#Labeling-Configuration) or speech labeling is a standard annotation technique that concerns separating sounds and labeling with specific metadata.
\

File: docs/source/guide/index.md [hunk 4]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Audio labeling or speech labeling is a standard annotation technique that concerns separating sounds and labeling with specific metadata.
+
+**Audio tagging**
+Audio tagging is a task to predict the tags of audio clips. This includes music tagging, acoustic scene classification, audio event classification, and so on.
+
+**Audio transcription**
```
Comment (anithamuthu2022) [inline docs/source/guide/index.md:73]: Audio transcription](https://labelstud.io/blog/improve-amazon-transcribe-audio-transcriptions-with-label-studio/) transforms any video or audio file into an entire experience for your audience. It is available in a written format of any verbal communication.

File: docs/source/guide/index.md [hunk 5]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Audio transcription transforms any video or audio file into an entire experience for your audience. It is available in a written format of any verbal communication.
+
+**Automatic Speech Recognition (ASR)**
+ASR refers to the problem of getting a program to automatically transcribe spoken language (speech-to-text).
+
+**Bounding box object detection**
+A bounding box describes the spatial location of an object. It is rectangular, which is determined by the "x" and "y" coordinates of the upper-left corner of the rectangle and such coordinates of the lower-right corner.
```
Comment (anithamuthu2022) [inline docs/source/guide/index.md:80]: A [bounding box](/image_bbox.html) describes the spatial location of an object. It is rectangular, which is determined by the "x" and "y" coordinates of the upper-left corner of the rectangle and such coordinates of the lower-right corner. For more information, see the [labeling configuration for object detection with bounding boxes](/image_bbox.html#Labeling-Configuration).

File: docs/source/guide/index.md [hunk 6]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+
+**Consolidated Receipt Dataset for Post-OCR Parsing (CORD)**
+COResets and Data Subset slection is an efficient and scalable library for making machine learning time, energy, cost, and compute efficient built on top of pytorch. It uses a subset of training data for model training and incorporates various state-of-the-art data selection strategies to select the proper representative data subset from massive datasets.
```
Comment (anithamuthu2022) [inline docs/source/guide/index.md:83]: [COResets and Data Subset selection](https://labelstud.io/blog/improve-ocr-quality-for-receipt-processing-with-tesseract-and-label-studio/) is an efficient and scalable library for making machine learning time, energy, cost, and compute efficient built on top of PyTorch. It uses a subset of training data for model training and incorporates various state-of-the-art data selection strategies to select the proper representative data subset from massive datasets.

File: docs/source/guide/index.md [hunk 7]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+ASR refers to the problem of getting a program to automatically transcribe spoken language (speech-to-text).
+
+**Bounding box object detection**
+A bounding box describes the spatial location of an object. It is rectangular, which is determined by the "x" and "y" coordinates of the upper-left corner of the rectangle and such coordinates of the lower-right corner.
+
+**Consolidated Receipt Dataset for Post-OCR Parsing (CORD)**
+COResets and Data Subset slection is an efficient and scalable library for making machine learning time, energy, cost, and compute efficient built on top of pytorch. It uses a subset of training data for model training and incorporates various state-of-the-art data selection strategies to select the proper representative data subset from massive datasets. 
+
+**Data annotation**
+Data annotation is the categorization and labeling of data for AI applications. It helps machines to learn the certain patterns and correlate the results, and the use the data sets to recognize the similar patterns in future to predict the results.
```
Comment (anithamuthu2022) [inline docs/source/guide/index.md:86]: [Data annotation](https://labelstud.io/blog/from-raw-data-to-a-trained-model-automate-your-ml-pipeline-with-label-studio-and-amazon-sagemaker/) is the categorization and labeling of data for AI applications. It helps machines to learn the certain patterns and correlate the results, and the use the data sets to recognize the similar patterns in future to predict the results.

File: docs/source/guide/index.md [hunk 8]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+
+**Data labeling**
```
Comment (makseq) [inline docs/source/guide/index.md:88]: Not sure there is a big difference between data annotation and labeling, better to combine them together and keep data labeling description

File: docs/source/guide/index.md [hunk 9]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Data labeling is the process of identifying raw data (images, text files, videos, and so on) and adding one or more meaningful and informative labels to provide context so that a machine learning model can learn from it.
+
+**Data parsing**
```
Comment (makseq) [inline docs/source/guide/index.md:91]: It's not related, better to remove

File: docs/source/guide/index.md [hunk 10]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+**Audio tagging**
+Audio tagging is a task to predict the tags of audio clips. This includes music tagging, acoustic scene classification, audio event classification, and so on.
+
+**Audio transcription**
+Audio transcription transforms any video or audio file into an entire experience for your audience. It is available in a written format of any verbal communication.
+
+**Automatic Speech Recognition (ASR)**
+ASR refers to the problem of getting a program to automatically transcribe spoken language (speech-to-text).
+
+**Bounding box object detection**
```
Comment (makseq) [inline docs/source/guide/index.md:79]: we can provide a link to the corresponding template

File: docs/source/guide/index.md [hunk 11]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+When you upload data to Label Studio, each item in the dataset becomes a labeling task. The following table defines the terms that appear throughout Label Studio.
+
+**Annotation**
+Annotation is the process of labeling data in various formats such as video, images, or text, so that machines can understand it. The output of a labeling task which was previously called **Completions**. 
+
+**Artificial Intelligence (AI)**
+AI is the intelligence demonstrated by machines, as opposed to the natural intelligence displayed by animals including humans.  It refers to the simulation of human intelligence in machines that are programmed to think like humans and mimic their actions.
+
+**Audio classification**
+Audio classification or sound classification can be referred to as the process of analyzing audio recordings. This amazing technique has multiple applications in the field of AI and data science such as chatbots, automated voice translators, virtual assistants, music genre identification, and text to speech applications.
```
Comment (makseq) [inline docs/source/guide/index.md:65]: we can provide a link to the corresponding template

File: docs/source/guide/index.md [hunk 12]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+**Data annotation**
+Data annotation is the categorization and labeling of data for AI applications. It helps machines to learn the certain patterns and correlate the results, and the use the data sets to recognize the similar patterns in future to predict the results.
+
+**Data labeling**
+Data labeling is the process of identifying raw data (images, text files, videos, and so on) and adding one or more meaningful and informative labels to provide context so that a machine learning model can learn from it.
+
+**Data parsing**
+Data parsing is a method of converting data from one format to another format. For example, converting from HTML to JSON.
+
+**Dialog processing**
```
Comment (makseq) [inline docs/source/guide/index.md:94]: It's something creepy, better to remove it

File: docs/source/guide/index.md [hunk 13]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Dialog processing or interactive document creation is the complementary method to batch processing and background processing. It is defined by the active user who creates all or part of the document in the system editor.
+
+**Docker**
```
Comment (makseq) [inline docs/source/guide/index.md:97]: It's out of our scope to write what docker is

File: docs/source/guide/index.md [hunk 14]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+**Document classification**
+Document classification is the act of labeling or tagging documents using categories, depending on their content.
+
+**Dataset**
+A dataset (or data set) is a collection of data comprised of individual items, or labeling tasks that you import into Label Studio.
+
+**Emotion recognition**
+Emotion recognition is the process of identifying human emotions.
+
+**Entity recognition**
```
Comment (makseq) [inline docs/source/guide/index.md:109]: better to replace it this named entity recognition - https://en.wikipedia.org/wiki/Named-entity_recognition - more suitable term for us

File: docs/source/guide/index.md [hunk 15]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Entity recognition is a new technology that can enable investigators and document review managers to get to the pertinent data faster.
+
+**Human labeling**
+Human labeling enables you to label images in your training dataset at scale.
```
Comment (makseq) [inline docs/source/guide/index.md:113]: why images? everything

File: docs/source/guide/index.md [hunk 16]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+
+**Image classification**
```
Comment (makseq) [inline docs/source/guide/index.md:115]: Usually image classification is about the whole image classification, not at pixel-level

File: docs/source/guide/index.md [hunk 17]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Image classification is the process of categorizing and labeling groups of pixels or vectors within an image based on specific rules.
+
+**Image labeling**
+Image labeling is the process of identifying and marking various details in an image. It is useful when automating the process of generating metadata or making recommendations to users based on details in their images.
+
+**Image segmentation**
+Image segmentation is the process of assigning a label to every pixel in an image such that pixels with the same label share certain characteristics.
+
+**Image tagging**
```
Comment (makseq) [inline docs/source/guide/index.md:124]: I don't feel the difference between tag and label, better to combine it with image labeling

File: docs/source/guide/index.md [hunk 18]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Image tagging is the process of labeling images based on figures within a certain picture. It enables the classification of visuals through the use of tags and labels.
+
+**Inventory tracking**
+Inventory tracking system allows you to label exact products by given brand names illustrated by relevant sample photo.
+
+**Label**
+A label is a key-value pair, stored as a string. You can specify multiple labels for an object, but each key must be unique within an object.
```
Comment (makseq) [inline docs/source/guide/index.md:131]: It's weird description for label term, better to remove label from here and put it into Label Studio Terms section

File: docs/source/guide/index.md [hunk 19]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+
+**Label in machine learning**
+In machine learning, a label is added by human annotators to explain a piece of data to the computer. This process is known as data annotation and is necessary to show the human understanding of the real world to the machines. Data labeling tools and providers of annotation services are an integral part of a modern AI project.
+
+**Label studio**
```
Comment (makseq) [inline docs/source/guide/index.md:136]: Label **S**tudio

File: docs/source/guide/index.md [hunk 20]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Label Studio is an open source data labeling tool for labeling and exploring multiple types of data.
+
+**Label studio enterprise**
```
Comment (makseq) [inline docs/source/guide/index.md:139]: **S**tudio **E**nterprise

File: docs/source/guide/index.md [hunk 21]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+**Machine learning**
+Machine learning is a branch of AI and computer science which focuses on the use of data and algorithms to imitate the way that humans learn, gradually improving its accuracy.
+
+**Machine Learning Model Operationalization Management (MLOps)**
+MLOps is a set of practices that aims to deploy and maintain machine learning models in production reliably and efficiently.
+
+**Model**
+A model in machine learning is the output of a machine learning algorithm run on data. A model represents what was learned by a machine learning algorithm.
+
+**Modeling**
```
Comment (makseq) [inline docs/source/guide/index.md:154]: better to remove it, it's something blurry

File: docs/source/guide/index.md [hunk 22]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Object detection relates to computer vision and image processing that deals with detecting instances of semantic objects of a certain class (such as humans, buildings, or cars) in digital images and videos.
+
+**Optical Character Recognition (OCR)**
+Optical character recognition (OCR) is a technology that analyzes the text of a page and turns the letters into code that may be used to process information.
+
+**Outliner**
+Outliner is the place where you can see all the details about annotation, regions, and labeling history. The Outliner and Details panel are both collapsible and detachable, so you can arrange them the way you want.
+
+**Persistent storage**
+Persistent storage is any data storage device that retains data after power to that device is shut off. It is also sometimes referred to as nonvolatile storage.
```
Comment (makseq) [inline docs/source/guide/index.md:176]: after power **off**?

File: docs/source/guide/index.md [hunk 23]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+
+**PostgreSQL**
+PostgreSQL, commonly pronounced “Post-GRE” is an open source database that has a strong reputation for its reliability, flexibility, and support of open technical standards.
+
+**Predictions**
```
Comment (makseq) [inline docs/source/guide/index.md:181]: In LS predictions and pre-annotations are the same, better to combine it with pre-annotations

File: docs/source/guide/index.md [hunk 24]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+**Pre-annotations** 
+Predicted annotations in Label Studio format, either in a file or from a machine learning backend. See [import pre-annotations](predictions.html).
+
+**Region**
+The portion of the task identified for labeling. For images, an example region is a bounding box. For text, an example region is a span of text. Often has a label assigned to it. 
+
+**Relation** 
+A defined relationship between two labeled regions. 
+
+**Result**
```
Comment (makseq) [inline docs/source/guide/index.md:193]: Result and region are very similar, I would say regions are subset of results, better to combine these two terms

File: docs/source/guide/index.md [hunk 25]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+**Semantic labeling**
+Semantic labeling, or semantic segmentation, involves assigning class labels to pixels. It models the predicate-argument structure of a sentence and is often described as answering "Who did what to whom"? For example, the task involves assigning at each pixel a label that is most consistent with local features at that pixel and with labels estimated at pixels in its context, based on consistency models learned from training data.
+
+**Server-side Request Forgery (SSRF)**
+SSRF is a web security vulnerability that allows an attacker to induce the server-side application to make requests to an unintended location.
+
+**Single Sign-on (SSO)**
+SSO is a technology which combines several application login screens into one.
+
+**SOCKS proxy**
```
Comment (makseq) [inline docs/source/guide/index.md:214]: we don't need this, it's unrelated

File: docs/source/guide/index.md [hunk 26]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+SOCKS is an internet protocol that exchanges network packets between a client and server through a proxy server. 
+
+**SQLite or PostgreSQL databases**
```
Comment (makseq) [inline docs/source/guide/index.md:217]: Keep sqlite only, postgres was above

File: docs/source/guide/index.md [hunk 27]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+**Time series data labeling**
+Time series data, which is also said to as time-stamped data, is a sequence of data points tabulated in an order of time. Select and label patterns in time series visualizations and other interactive diagrams with a single click.
+
+**Time series segmentation**
+Time series segmentation is a method of time-series analysis in which an input time-series is divided into a sequence of discrete segments in order to reveal the underlying properties of its source.
+
+**Time series**
+Time series data, also referred to as time-stamped data, is a sequence of data points indexed in time order.
+
+**Transport Layer Security (TLS)/Secure Sockets Layer (SSL)**
```
Comment (makseq) [inline docs/source/guide/index.md:250]: it's too broad, better to remove

File: docs/source/guide/index.md [hunk 28]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+TLS and SSL are cryptographic protocols that authenticate server-to-device data transfers.
+
+**Video**
```
Comment (makseq) [inline docs/source/guide/index.md:253]: too broad

File: docs/source/guide/index.md [hunk 29]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+Video is an electronic medium for the recording, copying, playback, broadcasting, and display of moving visual media.
+
+**Video data labeling**
+Data labeling is the process of identifying raw data (images, text files, videos, and so on.) and adding one or more meaningful and informative labels to provide context so that a machine learning model can learn from it. For example, labels might indicate whether a photo contains a bird or car, which words were uttered in an audio recording, or if an x-ray contains a tumor. Data labeling is required for a variety of use cases including computer vision, natural language processing, and speech recognition.
```
Comment (makseq) [inline docs/source/guide/index.md:257]: completely wrong description

File: docs/source/guide/index.md [hunk 30]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+
+**Video tagging**
+Video tagging is used to index and catalog videos based on content. It also Identifies a location in a video for future reference.
```
Comment (makseq) [inline docs/source/guide/index.md:260]: better to combine with video labeling

File: docs/source/guide/index.md [hunk 31]
```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
...
+
+**Visual genome**
```
Comment (makseq) [inline docs/source/guide/index.md:262]: it's too narrow term for us

**Review by makseq — CHANGES_REQUESTED** [review]
1. I would split these terms into 2 categories 
  a. Common terms
  b. Label Studio Terms - terms from LS interface and flow

2. We have these terms, they are mostly correct:
https://labelstud.io/guide/index.html#Label-Studio-terminology 

3. I would move (1b) to (2)

File: docs/source/guide/index.md (hunk 29 above)
Comment (anithamuthu2022) [reply docs/source/guide/index.md:257]: Deleting this term for now.

**Comment by anithamuthu2022** [comment]
@makseq I've addressed your feedback changes here. Feel free to merge this PR upon review/approval. Thank you!

File: docs/source/guide/index.md (hunk 14 above)
Comment (anithamuthu2022) [reply docs/source/guide/index.md:83]: Removed and replaced it with NER.

File: docs/source/guide/index.md (hunk 16 above)
Comment (anithamuthu2022) [reply docs/source/guide/index.md:89]: Image classification is the process of categorizing and labeling images based on specific rules.

**Pull request closed without merging**